doc/when-things-go-wrong.rst

```diff
@@ -142,13 +142,16 @@ or just to learn how the runtime works.
 
 Normally the debug progress messages go to standard output.
 If the ``--windowed`` option is used when bundling a Windows app,
-they are displayed as MessageBoxes.
+they are sent to any attached debugger. If you don't have or are not using a
+debugger the DebugView_ the free (beer) tool can be used to display such
+messages. It must be run before the application being bundled.
+
 For a ``--windowed`` Mac OS app they are not displayed.
 
 Remember to bundle without ``--debug`` for your production version.
 Users would find the messages annoying.
 
-
+.. _DebugView: https://docs.microsoft.com/en-us/sysinternals/downloads/debugview
```

Comment:
Please move this up to just below the paragraph referring to DebugView. Mind an empy line above and below.

Also min to restore the empty line whcih has been here :-.)

Reply:
Ok